src/main/scala/org/apache/mesos/chronos/scheduler/jobs/stats/JobStats.scala

```diff
@@ -562,4 +585,99 @@ class JobStats @Inject()(clusterBuilder: Option[Cluster.Builder], config: Cassan
       ConsistencyLevel.valueOf(config.cassandraConsistency())
     }
   }
+
+  private def pushNotificationToHook(hookurl: String,
+                                     port: Int,
+                                     id: Option[String],
+                                     timestamp: Option[java.util.Date],
+                                     jobName: Option[String],
+                                     jobOwner: Option[String],
+                                     jobSchedule: Option[String],
+                                     jobParents: Option[java.util.Set[String]],
```

Comment:
I would like to dependent / children jobs of the current job. Option: We can also indicate whether it would kick off a child (looking at invocationmap) if the job was successful. Would be great to see for logging purposes / detecting any failures in dependency management.